src/components/SplitButton/SplitButtonMenu/__tests__/test-utils.ts

```diff
@@ -0,0 +1,12 @@
+let rafSpy: jest.SpyInstance;
+
+export const mockRequestAnimationFrame = () => {
+  rafSpy = jest.spyOn(window, "requestAnimationFrame").mockImplementation((cb: FrameRequestCallback) => {
+    cb(0);
```

Comment:
why 0?

Reply:
typescript-wise - it need to get a number, so I'm just sending 0, doesn't really matter